test/Sentry.Serilog.Tests/SentrySerilogOptionsTests.cs

```diff
@@ -15,4 +15,11 @@ public void Ctor_MinimumEventLevel_Error()
         var sut = new SentrySerilogOptions();
         Assert.Equal(LogEventLevel.Error, sut.MinimumEventLevel);
     }
+
+    [Fact]
+    public void Ctor_EnableLogs_False()
```

Comment:
question: this test may be superfluous, as `EnableLogs` is declared in the "base" `SentryOptions` rather than the derived `SentrySerilogOptions`